### DISCUSSION TITLE
Discord themed GUI

### DISCUSSION THREAD

**Discussion by boylerplet** [maximmax42/Discord-CustomRP#92] [Ideas]
This application can be styled using a better Material type GUI, something similar to the discord color scheme can be used to make the application more user friendly.

**Comment by maximmax42** [comment]
Styling a WinForms app is a royal pain in the ass. Some controls are just not changing much, for example I can't find a way to change colors for date picker.
But, I was working on trying to make at least something coherent for the past couple days, so there will be dark mode in the next version. It's not perfect, but it's something.

**Reply by boylerplet** [reply]
That's a problem I think can be fixed. If you can point me to what was used to making this application gui, then I can research about it and try to find a solution.

**Reply by maximmax42** [reply]
C# WinForms.
